app/forms/feedback_form.rb

```diff
@@ -0,0 +1,9 @@
+class FeedbackForm < Reform::Form
+  property :url
+  property :email
+  property :comment
+  property :role
+  property :organisation
```

Comment:
Role and organisation are here but not in the administrate forms. Should they be?